docs/so101-ws/index.md

```
@@ -0,0 +1,389 @@
+  | Follower-Arm Axis   | Motor | Gear Ratio | Code          |
+  | ------------------- | :---: | :--------: | :-----------: |
+  | Base / Shoulder Pan |   1   |  1 / 345   | STS-3215-C047 |
+  | Shoulder Lift       |   2   |  1 / 345   | STS-3215-C047 |
+  | Elbow Flex          |   3   |  1 / 345   | STS-3215-C047 |
+  | Wrist Flex          |   4   |  1 / 345   | STS-3215-C047 |
+  | Wrist Roll          |   5   |  1 / 345   | STS-3215-C047 |
+  | Gripper             |   6   |  1 / 345   | STS-3215-C047 |
+
+To configure the motors for the leader arm:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)